### ISSUE TITLE
在windows上运行export_model.py模块时报错

### ISSUE THREAD

**Issue by jinghuyue** [yeyupiaoling/PaddlePaddle-DeepSpeech#136]
运行export_model.py模块后报了很多“E:\PycharmProjects\PaddlePaddle-DeepSpeech-master\venv\lib\site-packages\paddle\fluid\framework.py:5344: UserWarning: Skip loading for 'fc_6.w_0'. fc_6.w_0 expected a shape [2048, 758], but the received shape is [2048, 4330].
  warnings.warn(”这样的错误，请问是哪里有问题呢？

**Comment by smpkali** [comment]
-----------  Configuration Arguments -----------
augment_conf_path: ./conf/augmentation.json
batch_size: 16
learning_rate: 0.0005
max_duration: 20.0
mean_std_path: ./dataset/mean_std.npz
min_duration: 0.5
num_conv_layers: 2
num_epoch: 50
num_rnn_layers: 3
output_model_dir: ./models/param
pretrained_model: None
resume_model: None
rnn_layer_size: 1024
shuffle_method: batch_shuffle_clipped
test_manifest: ./dataset/manifest.test
test_off: False
train_manifest: ./dataset/manifest.train
use_gpu: False
vocab_path: ./dataset/zh_vocab.txt
------------------------------------------------
dataset/manifest.noise不存在，已经忽略噪声增强操作！
[2022-03-23 10:04:01.377413] 数据增强配置：{'type': 'speed', 'aug_type': 'audio', 'params': {'min_speed_rate': 0.9, 'max_speed_rate': 1.1, 'num_rates': 3}, 'prob': 1.0}
[2022-03-23 10:04:01.377489] 数据增强配置：{'type': 'shift', 'aug_type': 'audio', 'params': {'min_shift_ms': -5, 'max_shift_ms': 5}, 'prob': 1.0}
[2022-03-23 10:04:01.377542] 数据增强配置：{'type': 'volume', 'aug_type': 'audio', 'params': {'min_gain_dBFS': -15, 'max_gain_dBFS': 15}, 'prob': 1.0}
[2022-03-23 10:04:01.377762] 数据增强配置：{'type': 'specaug', 'aug_type': 'feature', 'params': {'W': 0, 'warp_mode': 'PIL', 'F': 10, 'n_freq_masks': 2, 'T': 50, 'n_time_masks': 2, 'p': 1.0, 'adaptive_number_ratio': 0, 'adaptive_size_ratio': 0, 'max_n_time_masks': 20, 'replace_with_zero': True}, 'prob': 1.0}
[2022-03-23 10:04:01.461630] 训练数据数量：13362

[2022-03-23 10:04:01.462344] 测试数据数量：26

!!! The CPU_NUM is not specified, you should set CPU_NUM in the environment variable list.
CPU_NUM indicates that how many CPUPlace are used in the current task.
And if this parameter are set as N (equal to the number of physical CPU core) the program may be faster.

export CPU_NUM=4 # for example, set CPU_NUM as number of physical CPU core which is 4.

!!! The default number of CPU_NUM=1.
已杀死
大佬，为甚cpu训练失败呢

**Comment by yeyupiaoling** [comment]
可能是内存不够，也可能是卡死了。CPU不好训练，容易卡死。如果你有非常多的CPU，可以试试执行CPU数量：export CPU_NUM=32

**Comment by yeyupiaoling** [comment]
@jinghuyue 你使用同一个数据字典导出的模型吗？需要使用同一个训练模型和数据字典才行的。

**Comment by jinghuyue** [comment]
是用了作者提供的模型和字典，依然报这个错误

---- 回复的原邮件 ----
发件人 ***@***.***>日期 2022年03月24日 09:00 收件人 ***@***.***>抄送至 ***@***.******@***.***>主题 Re: [yeyupiaoling/PaddlePaddle-DeepSpeech] 在windows上运行export_model.py模块时报错 (Issue #136)
@jinghuyue 你使用同一个数据字典导出的模型吗？需要使用同一个训练模型和数据字典才行的。

**Comment by yeyupiaoling** [comment]
这不应该啊，你重新试试全部替换
